diff --git a/chrome/app/app-Info.plist b/chrome/app/app-Info.plist
index 6fada0a2b1a1f..39c9a98bdadd1 100644
--- a/chrome/app/app-Info.plist
+++ b/chrome/app/app-Info.plist
@@ -5,7 +5,7 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
 	<key>CFBundleDisplayName</key>
-	<string>${EXECUTABLE_NAME}</string>
+	<string>${CHROMIUM_SHORT_NAME}</string>
 	<key>CFBundleDocumentTypes</key>
 	<array>
 		<dict>
diff --git a/chrome/app/chromium_strings.grd b/chrome/app/chromium_strings.grd
index cb212bf396c79..edd4bbe48816f 100644
--- a/chrome/app/chromium_strings.grd
+++ b/chrome/app/chromium_strings.grd
@@ -294,10 +294,10 @@ If you update this file, be sure also to update google_chrome_strings.grd. -->
         </then>
         <else>
           <message name="IDS_PRODUCT_NAME" desc="The Chrome application name" translateable="false">
-            Chromium
+            ChromiumFish
           </message>
           <message name="IDS_SHORT_PRODUCT_NAME" desc="The Chrome application short name." translateable="false">
-            Chromium
+            ChromiumFish
           </message>
         </else>
       </if>
diff --git a/chrome/app/theme/chromium/BRANDING b/chrome/app/theme/chromium/BRANDING
index f8363d5b294fe..22622d6a37f74 100644
--- a/chrome/app/theme/chromium/BRANDING
+++ b/chrome/app/theme/chromium/BRANDING
@@ -1,9 +1,9 @@
 COMPANY_FULLNAME=The Chromium Authors
 COMPANY_SHORTNAME=The Chromium Authors
-PRODUCT_FULLNAME=Chromium
-PRODUCT_SHORTNAME=Chromium
-PRODUCT_INSTALLER_FULLNAME=Chromium Installer
-PRODUCT_INSTALLER_SHORTNAME=Chromium Installer
+PRODUCT_FULLNAME=ChromiumFish
+PRODUCT_SHORTNAME=ChromiumFish
+PRODUCT_INSTALLER_FULLNAME=ChromiumFish Installer
+PRODUCT_INSTALLER_SHORTNAME=ChromiumFish Installer
 COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The Chromium Authors. All rights reserved.
 MAC_BUNDLE_ID=org.chromium.Chromium
 MAC_CREATOR_CODE=Cr24
